dslr/runner.py

```diff
@@ -62,15 +62,15 @@ def exec_sql(
     global pg_client
 
     if not pg_client:
-        # We always want to connect to the `postgres` and not the target
+        # We always want to connect to the `template1` and not the target
         # database because none of our operations need to query the target
         # database.
         pg_client = PGClient(
             host=settings.db.host,
             port=settings.db.port,
             user=settings.db.username,
             password=settings.db.password,
-            dbname="postgres",
+            dbname="template1",
```

Comment:
Would it make more sense to use `template0` here? That database should never be modified by users, but `template1` is intended to be modified by users to provide per-cluster defaults for new databases. It seems like it might be a valid use case to use `dslr` to snapshot your `template1` DB while experimenting with changes to it locally, whereas that should never be the case for `template0`.

Relevant docs: https://www.postgresql.org/docs/current/manage-ag-templatedbs.html

Reply:
Thanks for the review @bcdickinson . I agree. I can accept this PR if we can change it to use `template0`.